tools/debian/rules

```diff
@@ -43,3 +43,7 @@ override_dh_install:
 
 	dh_install -Xusr/src/debug
 	make install-tests DESTDIR=debian/cockpit-tests
+
+# will be the default in compat 13
```

Comment:
We already build-depend on `debhelper-compat (= 13)`, no? So aren't we already using compat level 13?

Reply:
Argh, you are right. I fished this out from #18112, but indeed it's not needed any more.

Reply:
Nice!